assets/readme.md

```diff
@@ -1,19 +1,121 @@
+
+[**Discover**](#Discover) new accounts that are similar to the profiles in any segment defined in Hull. Leverage our advanced segmentation engine to search through over 20M companies.
+
+The Clearbit connector supports to `create users`, `add traits` and `update traits`.
+
+## Find Your Clearbit API Key
+
+You can find your key in your Clearbit dashboard under the API menu link or by navigating to [https://dashboard.clearbit.com/api](https://dashboard.clearbit.com/api). Clearbit provides two keys, a *secret API key* and a *publishable API key*. Use the **secret API key** to integrate Hull with Clearbit:
+![Find your Clearbit API Key](./docs/clearbitapi01.png)
+
+## Enrichment
+
+The Enrichment feature of the connector allows you to complete a users profile with more than 85 data points provided by Clearbit’s API.
+
+**Configure Enrichment**
+You can enable or disable this feature by toggling the checkbox in the **Enrichment** section of the Settings tab:
+![Enrichment 1](./docs/enrichment01.png)
+
+Enrichment is enabled by default but will only result in API calls to Clearbit’s API if you specify at least one segment in the provided field. You can search segments by simply typing a part of the name into the field or by opening the dropdown via the triangular button.
+![Enrichment 2](./docs/enrichment02.png)
+
+**Which users do get enriched?**
+The connector only enriches users who have an *email address* *and have never been enriched before*. The connector sets the trait `clearbit/enriched_at` with the value of the timestamp if the user has been sent to Clearbit’s API.
+
+![Enrichment 3](./docs/enrichment03.png)
+
+Note: The connector will perform the enrich operation exactly only once, even if this API call does not yield additional data points.
+
+**When do users get enriched?**
+The connector attempts the enrichment as soon as the user enters one of the whitelisted segments. Please note that the segment needs to exist before the user gets created, otherwise this automatic enrichment will not be performed. 
+You can always enforce the enrichment by selecting one or multiple users on the users page and send them to your Clearbit connector via the “Actions” button. If you perform this route, the whitelisted segments will be ignored and all users that have an email address and no trait `clearbit/enriched_at` will be processed.
+
+Please note that users get only enriched ***exactly once***. Even if this enrichment does not yield any data, the Connector will not run another enrichment.
+
+**How long does it take to enrich users?**
+In most cases this process will take no longer than a couple of minutes. In some cases the Clearbit API marks a lookup as pending, which results in a delay of processing for up to an hour.
```

Comment:
@unity, this is not about presenting technical accurate information, this is about UX, and reality is that the loop to enrich a user in Hull is taking minutes, best case seconds but we cannot rely on that. This is about reducing the amount of tickets/questions and it seems that we got questions in the past, because @sbellity brought it up and TBass mentioned it as well. This addresses the perceived reality of the user and we want to avoid wrong expectations. @sbellity or can we guarantee immediate processing?